tutorials/Consistent_Scores.ipynb

```diff
@@ -0,0 +1,452 @@
+    "\n",
+    "1. produce forecasts (or develop a forecast system) that optimises the expected verification score, or\n",
+    "2. produce an honest forecast (or develop an honest forecast system) that is consistent with the forecast directive. \n",
+    "\n",
+    "Using a consistent scoring function to evaluate forecast performance ensures that objectives both 1 and 2 can be met without any tensions. For example, the Mean Squared Error (MSE) is a consistent score for forecasting the expected rainfall amount, since forecasting the \"expected rainfall amount\" will minimise the expected MSE. However the Mean Absolute Error (MAE) is not consistent with forecasting the expected amount and is instead consistent with forecasting the median value. Consistent scoring functions are formally defined in ([Gneiting et al., 2011](https://www.tandfonline.com/doi/abs/10.1198/jasa.2011.r10138)).\n",
+    "\n",
+    "\n",
+    "You may already know that the MSE is a consistent score for forecasting the mean and the MAE is a consistent score for forecasting the median. But did you know that there are a whole family of scores that are consistent for prediciting the mean, median, or quantiles? \n",
+    "\n",
+    "The consistent scoring module in `scores` provides access to these families of consistent scores that can be used to emphasise predictive performance across desired decision thresholds.\n",
+    "\n",
+    "### Example 1. A score consistent with the mean\n",
+    "Let's jump into an example that illustrates a scoring function that is consistent with predicting the mean value using `scores`. Suppose we want to evaluate the performance of expected daily precipitation forecasts, but we want to place increasing importance on correctly forecasting extreme values. \n",
+    "\n",
+    "\n",
+    "We want to used the `consistent_expectile_score` in `scores` to evaluate the expected daily precipitation forecasts. Note that medians relate to quantiles in an analogous way to how the mean relates to [expectiles](https://en.wikipedia.org/wiki/Expectile).\n",
+    "\n",
+    "Our consistent scoring function for expectiles is \n",
+    "\n",
+    " $$ S(x, y) =\n",
+    "    \\begin{cases}\n",
+    "    (1 - \\alpha)(\\phi(y) - \\phi(x) - \\phi'(x)(y-x)), & y < x \\\\\n",
+    "    \\alpha(\\phi(y) - \\phi(x) - \\phi'(x)(y-x)), & x \\leq y\n",
+    "    \\end{cases}\n",
+    "$$\n",
+    "\n",
+    "Where:\n",
+    "- $x$ is the forecast\n",
+    "- $y$ is the observation\n",
+    "- $\\alpha$ is the expectile level\n",
+    "- $\\phi$ is a [convex function](https://en.wikipedia.org/wiki/Convex_function) of a single variable\n",
+    "- $\\phi'$ is the [subderivative](https://en.wikipedia.org/wiki/Subderivative) of $\\phi$. The subderivative is a generalisation of the derivative for convex functions and coincides with the derivative when the convex function is differentiable\n",
+    "- $S(x,y)$ is the score.\n",
```

Comment:
I wonder whether it is worth stating somewhere, that taking alpha0.5 and phi(z)=2t^2 (please check the constant!) gives S(x,y)=(x-y)^2, with the interpretation that squared loss places equal weights on each decision threshold since phi''(z) is constant.